lib/bundle-validator.js

```diff
@@ -106,6 +107,119 @@ function validateThemeConfiguration(callback) {
     callback(null, true);
 }
 
+/**
+ * Find translatable strings
+ * @param {object} schema
+ * @param {function} callback
+ */
+function getTranslatableStrings(schema, callback) {
+    var trackedKeys = ['name', 'content', 'label', 'settings', 'options'];
+
+    schema.forEach(element => {
+        Object.keys(element).forEach(key => {
+            const value = element[key];
+
+            if (trackedKeys.indexOf(key) === -1) {
+                return;
+            }
+
+            if (Array.isArray(value)) {
+                return getTranslatableStrings(value, callback);
+            }
+
+            callback(value);
+        });
+    });
+}
+
+/**
+ * Find i18n keys in schema
+ * @param {object} schema
+ * @return {array}
+ */
+function getI18nKeys(schema) {
+    var keys = [];
```

Comment:
Lets review all usage of var in this PR and see if we can use const in each case.